.swiftlint.yml

```diff
@@ -0,0 +1,48 @@
+
+# Exclude deprecated/legacy services
+excluded: # paths to ignore during linting. Takes precedence over `included`.
+  - Source/AlchemyDataNewsV1
+  - Source/AlchemyLanguageV1
+  - Source/AlchemyVisionV1
+  - Source/DialogV1
+  - Source/PersonalityInsightsV2
+  - Source/RelationshipExtractionV1Beta
+  - Source/TradeoffAnalyticsV1
+  - Source/SupportingFiles/Dependencies
+  - Tests/AlchemyDataNewsV1Tests
+  - Tests/AlchemyLanguageV1Tests
+  - Tests/AlchemyVisionV1Tests
+  - Tests/DialogV1Tests
+  - Tests/PersonalityInsightsV2Tests
+  - Tests/RelationshipExtractionV1BetaTests
+  - Tests/TradeoffAnalyticsV1Tests
+
+disabled_rules: # rule identifiers to exclude from running
+  - closure_parameter_position
+  - empty_parentheses_with_trailing_closure
+  - opening_brace
+  - redundant_string_enum_value
+  - todo
+  - type_body_length
```

Comment:
Thanks for the careful documentation on all these options. 

*Empty Parentheses with Trailing Closure*: I disabled this simply because of the shear number of current violations -- 143.  But I agree that it's probably better to adopt the preferred style, so I'll make this change.  And happily, it turns out that the 143 warnings were really 11 instances within RestKit that were each counted 13 times because of the symlinks.  So this was much less invasive than expected to fix.

*todo*: I disabled this thinking that we wanted to get to zero warnings.  But maybe it is better to leave these as warning as you suggest.  I'll make that change.

*type_body_length*: This one I'm not sure about.  The most egregious violator is currently `Conversation.java`, with 1315 lines.  This service is generated, and the size of the type body reflects our decision to put all service methods in a single service class.  That just naturally produces large type bodies.  So adopting this restriction would be counter to our design for generated services.